pkgs/development/compilers/vala/default.nix

```diff
@@ -59,6 +53,12 @@ in rec {
     sha256  = "16cjybjw100qps6jg0jdyjh8hndz8a876zmxpybnf30a8vygrk7m";
   };
 
+  vala_0_36 = generic {
```

Comment:
What depends on `vala_0_36`?

Reply:
Hmm, you are right, I must have confused it with 0.26.